docs/source-app/index.rst

```diff
@@ -216,17 +216,18 @@ Keep Learning
    :caption: How to...
 
    Access the App State <workflows/access_app_state/access_app_state>
-   Add a web user interface (UI) <workflows/add_web_ui/index>
    Add a web link  <workflows/add_web_link>
+   Add a web user interface (UI) <workflows/add_web_ui/index>
    Add encrypted secrets <glossary/secrets>
    Arrange app tabs <workflows/arrange_tabs/index>
+   Cache Work run calls  <workflows/run_work_once>
+   Customize your cloud compute <core_api/lightning_work/compute>
    Develop a Command Line Interface (CLI) <workflows/build_command_line_interface/index>
    Develop a Lightning App <workflows/build_lightning_app/index>
    Develop a Lightning Component <workflows/build_lightning_component/index>
    Develop a REST API <workflows/build_rest_api/index>
-   Cache Work run calls  <workflows/run_work_once>
-   Customize your cloud compute <core_api/lightning_work/compute>
    Extend an existing app <workflows/extend_app>
+   Mount Cloud Data <workflows/mount_cloud_object_store>
```

Comment:
Let's maybe make a more human-readable how to.

Something like ``How to add datasets to your works``

Reply:
This was decided upon by @edenlightning

Reply:
Maybe "Add cloud data"?

This isn't specifically for datasets, and is only applicable when running on the cloud